test/chaos/run-chaos.sh

```
@@ -0,0 +1,153 @@
+RUN_DIR=".chaos-run"
+IMAGE="indigopay-backend-test:ci"
+
+cd "$(dirname "$0")/../.." # repo root
+
+log() { echo -e "\n\033[1m[chaos-host] $*\033[0m"; }
+
+cleanup() {
+  log "tearing down chaos topology"
+  $COMPOSE down -v --remove-orphans >/dev/null 2>&1 || true
+  rm -rf "$RUN_DIR" 2>/dev/null || sudo rm -rf "$RUN_DIR" 2>/dev/null || true
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)